de_de.lang

```diff
@@ -244,3 +244,61 @@ gui.xaero_delete_world_msg2=Dies kann nicht rückgängig gemacht werden. Bitte p
 gui.xaero_delete_multi_world=Lösche Sub-Welt
 gui.xaero_delete_multi_world_msg1=Möchtest du alle Wegpunkt-Daten für die gewählte Sub-Welt löschen?
 gui.xaero_delete_multi_world_msg2=Dies kann nicht rückgängig gemacht werden. Bitte prüfe noch einmal, dass du das tun willst.
+gui.xaero_centered_enlarged=Beim Vergrößern zentrieren
+gui.xaero_waypoints_distance_scale=Wegpunktdistanz Textgröße
+gui.xaero_waypoints_name_scale=Wegpunktname Textgröße
+gui.xaero_waypoint_distance_visibility_angle=WP Dist. Horiz. Winkel
+gui.xaero_waypoint_distance_vertical_visibility_angle=WP Dist. Vertik. Winkel
+gui.xaero_always_show_distance=WP Entf. immer anzeigen
+gui.xaero_show_light_level=Lichtlevel Anzeige
+gui.xaero_show_angles=Blickwinkel anzeigen
+gui.xaero_biomes_vanilla=Biome im Vanilla Farbmodus
+gui.xaero_show_dimension_name=Akuelle Dimension anzeigen
+gui.xaero_show_time=Spiel Uhrzeit anzeigen
+gui.xaero_minimap_text_align=Info Text Ausrichtung
+gui.xaero_center=Mitte
+gui.xaero_left=Links
+gui.xaero_right=Rechts
+gui.xaero_main_entity_as=Spielersymbol als
+gui.xaero_arrow=Pfeil
+gui.xaero_dot=Punkt
+gui.xaero_crosshair=Fadenkreuz
+gui.xaero_cave_maps_depth=Höhlenmodus Tiefe
+gui.xaero_cross_tp=Dimensionsübergreif. TP
+gui.xaero_player_arrow_opacity=Pfeil Deckkraft
+gui.xaero_zoomed_out_enlarged=Herauszoomen beim Vergrößern
+gui.xaero_teleport_default_command=Standard Teleport Befehl
+gui.xaero_waypoints_bottom=Neue WegPunkte oben
+gui.xaero_dot_name_scale=Name Punkt Größe
+gui.xaero_entity_heads_scale=Objekt Symbol Größe
+gui.xaero_dots_style=Objekt Punkte Größe
+gui.xaero_dots_style_default=Standard
+gui.xaero_dots_style_legacy=Vererbt
+gui.xaero_always_entity_nametags=Objektnametags jederz. anzeig.
```

Comment:
Please translate what this means for me. I'm not sure if it's correct.

Reply:
I have shortened it because of the Button Sizes etc.
"Objektnametags jederzeit anzeigen"